package.json

```
@@ -27,51 +27,73 @@
     "url": "https://github.com/jupyterlab-contrib/jlab-enhanced-launcher.git"
   },
   "scripts": {
-    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
+    "build": "jlpm build:lib && jlpm build:labextension:dev",
     "build:labextension": "jupyter labextension build .",
     "build:labextension:dev": "jupyter labextension build --development True .",
-    "build:lib": "tsc",
-    "build:prod": "jlpm run build:lib && jlpm run build:labextension",
-    "clean": "jlpm run clean:lib",
-    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
-    "clean:labextension": "rimraf jlab_enhanced_launcher/labextension",
+    "build:lib": "tsc --sourceMap",
+    "build:lib:prod": "tsc",
+    "build:prod": "jlpm clean && jlpm build:lib:prod && jlpm build:labextension",
+    "clean": "jlpm clean:lib",
+    "clean:all": "jlpm clean:lib && jlpm clean:labextension && jlpm clean:lintcache",
+    "clean:labextension": "rimraf jlab_enhanced_launcher/labextension jlab_enhanced_launcher/_version.py",
     "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
-    "eslint": "eslint . --ext .ts,.tsx --fix",
-    "eslint:check": "eslint . --ext .ts,.tsx",
+    "clean:lintcache": "rimraf .eslintcache .stylelintcache",
+    "eslint": "jlpm eslint:check --fix",
+    "eslint:check": "eslint . --cache --ext .ts,.tsx",
     "install:extension": "jlpm build",
-    "prepare": "jlpm run clean && jlpm run build:prod",
+    "lint": "jlpm stylelint && jlpm prettier && jlpm eslint",
+    "lint:check": "jlpm stylelint:check && jlpm prettier:check && jlpm eslint:check",
+    "prettier": "jlpm prettier:base --write --list-different",
+    "prettier:base": "prettier \"**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}\"",
+    "prettier:check": "jlpm prettier:base --check",
+    "stylelint": "jlpm stylelint:check --fix",
+    "stylelint:check": "stylelint --cache \"style/**/*.css\"",
     "watch": "run-p watch:src watch:labextension",
     "watch:labextension": "jupyter labextension watch .",
-    "watch:src": "tsc -w"
+    "watch:src": "tsc -w --sourceMap"
   },
   "dependencies": {
-    "@jupyterlab/application": "^3.0.0",
-    "@jupyterlab/apputils": "^3.0.0",
-    "@jupyterlab/coreutils": "^5.0.0",
-    "@jupyterlab/launcher": "^3.0.0",
-    "@jupyterlab/settingregistry": "^3.0.0",
-    "@jupyterlab/statedb": "^3.0.0",
-    "@jupyterlab/ui-components": "^3.0.0",
-    "@lumino/algorithm": "^1.3.3",
-    "@lumino/coreutils": "^1.5.3",
-    "@lumino/widgets": "^1.16.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "@jupyterlab/application": "^4.3.5",
+    "@jupyterlab/apputils": "^4.4.5",
+    "@jupyterlab/coreutils": "^6.3.5",
+    "@jupyterlab/filebrowser": "^4.3.5",
+    "@jupyterlab/launcher": "^4.3.5",
+    "@jupyterlab/settingregistry": "^4.3.5",
+    "@jupyterlab/statedb": "^4.3.5",
+    "@jupyterlab/ui-components": "^4.3.5",
```

Let's be less strict about the version here:

```suggestion
    "@jupyterlab/application": "^4.0.0",
    "@jupyterlab/apputils": "^4.0.0",
    "@jupyterlab/coreutils": "^6.0.0",
    "@jupyterlab/filebrowser": "^4.0.0",
    "@jupyterlab/launcher": "^4.0.0",
    "@jupyterlab/settingregistry": "^4.0.0",
    "@jupyterlab/statedb": "^4.0.0",
    "@jupyterlab/ui-components": "^4.0.0",
```